packages/webapp/src/journey-validation/allocation/applicant-details.js

```diff
@@ -196,6 +227,22 @@ const applicantDetailsJourneys = [
   notAgentNotLandownerOrganisationJourney
 ]
 
+const applicantDetailsRouteDefinitions = [
+  AGENT_ACTING_FOR_CLIENT,
+  CHECK_DEFRA_ACCOUNT_DETAILS,
+  IS_LANDOWNER,
+  INDIVIDUAL_ORGANISATION,
+  CLIENT_INDIVIDUAL_ORGANISATION,
+  CLIENT_INDIVIDUAL_NAME,
+  CLIENT_ORGANISATION_NAME,
+  WRITTEN_AUTHORISATION_UPLOAD,
+  WRITTEN_AUTHORISATION_CHECKED,
+  LANDOWNER_CONSENT_UPLOAD,
+  LANDOWNER_CONSENT_CHECK,
+  PROOF_OF_PERMISSION
+]
+
 export {
-  applicantDetailsJourneys
+  applicantDetailsJourneys,
+  applicantDetailsRouteDefinitions
```

Comment:
Expose all route definitions because some might not be exposed with the journeys. Or may be exposed multiple times and in a nested array.